src/ripple/shamap/SHAMapItem.h

```diff
@@ -40,6 +40,7 @@ class SHAMapItem
 public:
     SHAMapItem (uint256 const& tag, Blob const & data);
     SHAMapItem (uint256 const& tag, Serializer const& s);
+    SHAMapItem (uint256 const& tag, Serializer&& s);
```

Comment:
True.  But the motivation to prefer one over the other is low.  Our choices are 1 signature and an extra vector move construction vs 2 signatures.  I can't really get too worked up over either choice.